test/server/config_validation/cluster_manager_test.cc

```diff
@@ -0,0 +1,58 @@
+#include "envoy/json/json_object.h"
```

Comment:
It occurred to me that another way you could get coverage is to write sufficient example configs to exercise all paths, but I think what you have here is fine if it doesn't regress coverage.